### ISSUE TITLE
[BUG] [typescript-axios] a property 'delete' is changed into '_delete' in output

### ISSUE THREAD

**Issue by joerg-walter-de** [OpenAPITools/openapi-generator#7474]
I have:

`    AccessRights_SingleRightsObject:
      description: 
      type: object
      additionalProperties: false
      properties:
        create:
          type: boolean
          default: false
          example: true
        read:
          type: boolean
          default: false
          example: true
        update:
          type: boolean
          default: false
          example: true
        delete:
          type: boolean
          default: false
          example: true
`

The output is:

`export interface AccessRightsSingleRightsObject {
    /**
     * @type {boolean}
     * @memberof AccessRightsSingleRightsObject
     */
    create?: boolean;
    /**
     * @type {boolean}
     * @memberof AccessRightsSingleRightsObject
     */
    read?: boolean;
    /**
     * @type {boolean}
     * @memberof AccessRightsSingleRightsObject
     */
    update?: boolean;
    /**
     * @type {boolean}
     * @memberof AccessRightsSingleRightsObject
     */
    _delete?: boolean;
}
`

**Comment by wing328** [comment]
I think it's because `delete` is a reserved word in typescript/axios.

**Comment by louis-lau** [comment]
Reserved words don't apply in this context:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Lexical_grammar#Reserved_word_usage

**Comment by sarumont** [comment]
@louis-lau In come contexts they may. The `LowercaseLambda` seems to respect the language's reserved words:

```java
    @Override
    public void execute(Template.Fragment fragment, Writer writer) throws IOException {
        String text = fragment.execute().toLowerCase(Locale.ROOT);
        if (generator != null && generator.reservedWords().contains(text)) {
            text = generator.escapeReservedWord(text);
        }
        writer.write(text);

    }
```

@wing328 Would a `lowercaseRaw` lambda (and corresponding `camelcaseRaw` and `pascalcaseRaw`) be an acceptable way to provide these? I just hit this myself in a custom generator which builds Express mappings, and `app.delete()` is the method to map an HTTP `DELETE` route. :)

**Comment by hatinena** [comment]
If someone stumbles upon this thread, solution is found here
https://github.com/OpenAPITools/openapi-generator/issues/7100
**tldr** use this option in your generate command :
--reserved-words-mappings delete=delete
